sharktank/sharktank/models/clip/testing.py

```diff
@@ -88,11 +83,8 @@ def export_clip_toy_text_model_iree_test_data(
     expected_last_hidden_state_output_path: PathLike,
 ):
     reference_config = clip_toy_text_model_config(reference_dtype)
-    input_ids = make_random_input_token_sequences(
-        batch_size=batch_size, config=reference_config
-    )
-    reference_theta = make_clip_text_model_random_theta(reference_config)
-    reference_model = ClipTextModel(theta=reference_theta, config=reference_config)
+    reference_model = ClipTextModel(config=reference_config)
+    input_ids = reference_model.sample_inputs(batch_size=batch_size)[1]["input_ids"]
```

Comment:
Will this be the canonical method of supplying input arguments, by utilizing model class attributes? I imagine it would be more flexible to allow the model config to define input shapes/dtypes, but that could be a follow-up patch if we want such functionality.

Reply:
I see that your example has `"my_model_arg": 5,` which I assume is what I was asking about. Disregard

Reply:
I view the input shape as a notion derived from other higher level config arguments. That is why I added the `sample_inputs` method. With it you can interrogate the model about the shape as well.
I was thinking that it should also provide a plausible sample from the expected input distribution, but this is probably too big of an ask. At least it should provide a random sample.